python/ray/air/tests/test_integration_mlflow.py

```diff
@@ -42,14 +42,16 @@ def clear_env_vars():
 
 class MLflowTest(unittest.TestCase):
     def setUp(self):
-        self.tracking_uri = tempfile.mkdtemp()
-        self.registry_uri = tempfile.mkdtemp()
+        self.tracking_uri = "sqlite:///" + tempfile.mkdtemp() + "/mlflow.sqlite"
+        self.registry_uri = "sqlite:///" + tempfile.mkdtemp() + "/mlflow.sqlite"
```

Comment:
why is this change to `sqlite:///` required in this PR?

Reply:
mlflow has been updated from 1.x to 2.x (2.x is not built for Py 3.7). There's been a breaking API change in that version switch, in that the local file backend now creates random experiment IDs instead of sequential ones starting from 0. The sqlite backend still starts from 0 (though this now points to a default experiment - thus the experiments we create in the tests start at ID 1). In order to keep changes to the tests minimal, switching the backend was an easy solution.

Reply:
We shipped a later mlflow version in previous Ray releases for python 3.8+ and some of our customers/users rely on it. Pinning it to the lower version here would downgrade it in these images.

Reply:
I can separate this out though to make it clear. We will just have some nightly dockers with an older mlflow built then.

Reply:
I've separated the test change and will re-upgrade mlflow in a follow-up PR